test/test-files/time.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$id": "https://example.com/time.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "A schema to test the TIME logical types in Parquet",
+  "type": "object",
+  "properties": {
+    "time_millis_field": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time field represented as a string with date-time format"
+        },
+        "unit": {
+          "type": "string",
+          "enum": ["MILLIS", "MICROS", "NANOS"],
+          "description": "The unit for the time value"
```

Comment:
How we expect units be defined a string or enum?

Reply:
I am fine with the enum or string, so what it is now is good by me.